packages/next-templates/src/components/ExampleHeader/HeaderIngelogd/HeaderIngelogd.tsx

```diff
@@ -0,0 +1,103 @@
+'use client';
+
+import { Button, Link, PageHeader, Textbox } from '@utrecht/component-library-react';
+import { PopOverButton } from '../../PopOver/PopOverButton';
+import { PopOverDialog } from '../../PopOver/PopOverDialog';
+import Search from '../app/styling/assets/searchbar-icon.svg';
+import Xbold from '../app/styling/assets/x-bold.svg';
+import User from '../app/styling/assets/user-icon.svg';
+import React, { HTMLAttributes } from 'react';
+import { PageHeaderLogo } from '../../PageHeaderLogo';
+import '@utrecht/component-library-css';
+import './header.css';
+import './header.scss';
+
+interface ExampleHeaderProps extends HTMLAttributes<HTMLDivElement> {
+  search?: boolean;
+}
+
+export const HeaderIngelogd = ({ search }: ExampleHeaderProps) => (
+  <PageHeader className="example--header-home-page">
+    <div className="example-page-header__content">
+      <PopOverButton
+        aria-label="collapse menu navigation"
+        className="example--header-burger-menu-button"
```

Comment:
Van Robbert op mijn vorige PR heb ik de feedback ontvangen dat dit het BEM systeem is:

`[prefix]-[block]__[element]--[modifier]`

Als ik het goed begrijp zou je deze class via dat systeem dan beter bijv. `voorbeeld-header__burger-menu-button` kunnen noemen.